lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -179,756 +179,763 @@
       "gather": true
     },
     {
-      "startTime": 0,
+      "startTime": 1,
```

Comment:
I update these timings "manually" without running `update-report-fixtures.js`

Reply:
FYI the script accepts a parameter to only update a specific artifact. `yarn update:sample-artifacts Timing`